src/design-system/color/ColorMode.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { createContext, ReactNode, useContext, useMemo } from 'react';
 import { ColorMode, Palette, palettes } from './palettes';
 
+export type { ColorMode };
```

Comment:
Don't we typically put things like that at the bottom?

Reply:
I felt like it would feel awkward being at the bottom, if devs came to this file looking for `ColorMode`, they'd understand it is being imported from `./palettes` and exported accordingly without scrolling down.
If it is a code standard though, I'm happy to move it down!

Reply:
im fine with this since its not the main repo